Resources/public/inlineDeleteCheckboxes.js

```diff
@@ -0,0 +1,32 @@
+/*
+
+ This file is part of the Sonata package.
+
+ (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+ */
+
+var handleInlineDeleteCheckboxes = function() {
+    var checkboxes = jQuery('.sonata-admin-type-delete-checkbox');
+
+    if(checkboxes.length) {
+        var eventType = window.SONATA_CONFIG.USE_ICHECK ? 'ifChanged': 'change';
+
+        checkboxes.on(eventType, function() {
+            var id = jQuery(this).prop('id');
+
+            jQuery('*[id^=' + id.split('__')[0] + ']')
```

Comment:
> you don't need this. 

Absolutely bad habit of mine

> This is a very slow selector.

I know but couldn't come up with anything better. Tried it on a form with 140 inputs and it was ok